<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushclear1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar dados do cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushclear2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar dados do pedido</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>170</y>
      <width>1231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Agenda Personalizada</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Caixa de Páscoa</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>280</y>
      <width>195</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Caixa Surpresa de Aniversario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>211</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Data de Entrega:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_data">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>211</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar pedido no arquivo &quot;pedidos.txt&quot;</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>460</y>
      <width>1231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PEDIDO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CLIENTE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ITEM DO PEDIDO:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fechar programa</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Caneca</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>280</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>90</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_nome">
       <property name="text">
        <string>Nome do Cliente:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_nome"/>
     </item>
     <item>
      <widget class="QLabel" name="label_telefone">
       <property name="text">
        <string>Telefone do Cliente:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_telefone"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>541</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_endereco">
       <property name="text">
        <string>Endereço do Cliente:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_endereco"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>181</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_temacaneca">
       <property name="text">
        <string>Tema da Caneca:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_temacaneca"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>211</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_capa">
       <property name="text">
        <string>Capa da Agenda Personalizada:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_capa"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>241</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_pascoa">
       <property name="text">
        <string>Conteúdo da Caixa de Páscoa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_pascoa"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>231</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_aniver_2">
       <property name="text">
        <string>Nome e idade do aniversariante:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_aniver1"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>370</y>
      <width>231</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_aniver">
       <property name="text">
        <string>Tema da Caixa Surpresa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_aniver2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVer_Pedidos"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionVer_Pedidos">
   <property name="text">
    <string>Ver Pedidos</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
